README.md

```diff
@@ -4,19 +4,19 @@ Barcode scanning
 Context 
 -----------
 
-This application demonstrates how to scan a barcode on Coppernic device.
+This application demonstrates how to scan a barcode on a Coppernic device.
 
 Prerequisites
 -------------
 
-CpcSystemServices from version 2.0.4 needs to be installed on device.
+CpcSystemServices from version 2.0.4 needs to be installed on the device.
```

Comment:
Pas d'accord pour le `the`, est-ce que tu es sur de toi Kamel ?

Reply:
D'accord avec toi, bien vu ;-)